src/workerd/server/workerd.capnp

```diff
@@ -509,6 +511,8 @@ struct Worker {
 
   # TODO(someday): Support distributing objects across a cluster. At present, objects are always
   #   local to one instance of the runtime.
+
+  unwrapWith @13 :Text;
```

Comment:
I think it should be possible to specify this on a per-entrypoint basis.

Does it make sense to be able to specify multiple wrappers on a single entrypoint? I guess it depends on whether the wrapper is able to detect its own events and pass others through unmodified. I can imagine use cases where that's what we want, and other use cases where the wrapper wants to completely own the entrypoint. I guess let's start out supporting only one wrapper and see where it goes. People can always export multiple entrypoints to support multiple event types (though we still need to support alternate entrypoints on Cloudflare...).

Reply:
Perhaps the idea of multiple wrappers makes sense only in the case that we have a system for tagging requests with their type, and matching the appropriate wrapper for that type.

But on thinking about it a while, I don't think we want or need an explicit notion of event type tagging. I think it's probably better to use multiple entrypoints to support multiple event types.

Reply:
Ah, I see my other comment was mostly a duplicate of this, sorry.

I do think that if we want to start using this for new "official" event handlers then the desire for multiple wrappers on a single entrypoint will come up pretty quickly, if only to ensure that a script can export the custom event and still export `fetch`.